content/docs/user_guide/compatibility/_index.md

```diff
@@ -35,4 +36,5 @@ The following applications/images have been tested:
 *   tomcat
 *   wordpress
 
-[bug]: https://github.com/google/gvisor/issues
+[bug]: https://github.com/google/gvisor/issues/new?labels=area:%20compatibility,bug
```

Comment:
According to https://help.github.com/en/articles/applying-labels-to-issues-and-pull-requests, you need write access to the repo to add labels, so this will likely be a no-op for most users.

Reply:
"This" being the labels= component that sets the default labels.

Reply:
Ah, right. I had forgotten about that. I suppose a hint at the title would be more appropriate.